crates/sui-indexer-alt-graphql/src/api/types/transaction.rs

```diff
@@ -59,6 +63,21 @@ impl Transaction {
 
 #[Object]
 impl TransactionContents {
+    /// This field is set by senders of a transaction block. It is an epoch reference that sets a deadline after which validators will no longer consider the transaction valid. By default, there is no deadline for when a transaction must execute.
```

Comment:
nit, naming -- we no longer use the term "transaction block", just "transaction".


```suggestion
    /// This field is set by senders of a transaction. It is an epoch reference that sets a deadline after which validators will no longer consider the transaction valid. By default, there is no deadline for when a transaction must execute.
```